seq2science/rules/configuration_generic.smk

```diff
@@ -66,7 +68,8 @@ assert sorted(config['fqext'])[0] == config['fqext1'], \
 
 # read the config.yaml (not the profile)
 with open(workflow.overwrite_configfiles[0], 'r') as stream:
-    user_config = yaml.safe_load(stream)
+    # safe_load() with exception on duplicate keys
+    user_config = yaml.load(stream, Loader=UniqueKeyLoader)
```

Comment:
perhaps catch the error and pass it to the logger?

Reply:
that's what happens in the Loader!